examples/dh-5-example/build.gradle

```diff
@@ -1,5 +1,6 @@
 buildscript {
     repositories {
+        maven {url 'http://distro.marklogic.com/nexus/repository/maven-snapshots/'}

```

Comment:
This is only for beta releases and won't work for anyone outside of the VPN - our example projects (per their README) assume that the user has published a local copy of Gradle in case testing=true.